esmvaltool/utils/draft_release_notes.py

```diff
@@ -41,26 +41,36 @@
 }
 
 PREVIOUS_RELEASE = {
-    'esmvalcore': datetime.datetime(2021, 11, 8, 00),
-    'esmvaltool': datetime.datetime(2021, 11, 9, 00),
+    'esmvalcore': datetime.datetime(2022, 7, 15, 18),
+    'esmvaltool': datetime.datetime(2022, 7, 22, 18),
 }
+
+
 LABELS = {
```

Comment:
Would it be needed to define a title for some of the labels introduced in this PR, e.g. iris, UX?

Reply:
I don't know, half of the labels don't have a title anyway.